wire/service.go

```diff
@@ -0,0 +1,91 @@
+package wire
+
+import (
+	"context"
+	"net"
+	"strconv"
+
+	"github.com/evergreen-ci/mrpc"
+	"github.com/evergreen-ci/mrpc/mongowire"
+	"github.com/evergreen-ci/mrpc/shell"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/recovery"
+	"github.com/mongodb/jasper"
+	"github.com/pkg/errors"
+)
+
+// TODO: support jasper.RemoteClient functionality
```

Comment:
is there a ticket for this?

Reply:
https://jira.mongodb.org/browse/MAKE-1000